packages/transaction-controller/src/TransactionController.ts

```diff
@@ -1109,6 +1115,84 @@ export class TransactionController extends BaseController<
     return this.nonceTracker.getNonceLock(address);
   }
 
+  async approveTransactionsWithSameNonce(
+    listOfTxParams: TransactionParams[] = [],
+  ) {
+    if (listOfTxParams.length === 0) {
+      return '';
```

Comment:
For audit sake, I don't think returning an empty raw transaction is the cleanest API, but can see we're copying the extension implementation exactly for stability and backwards compatibility.

Reply:
Exactly, I also think that returning an empty string makes no sense here. We will eventually make this more meaningful in future anyway.